module/configs/pych.json

```diff
@@ -102,7 +102,7 @@
             ],
 
             "commands": [
-                "chpl %(cflags)s %(ctargets)s -o __TMP_PATH__",
+                "chpl %(cflags)s __DEPENDS__ %(ctargets)s -o __TMP_PATH__",
```

Comment:
That's fair, though configs do feel more symptomatic of a complete Chapel program.